src/controllers/solanaTransactionClient.ts

```diff
@@ -0,0 +1,41 @@
+import { appConfig } from "../config"
+import { SolanaMetadata, SolanaTransaction } from "../models/solanaTransaction"
+
+export interface SolanaTransactionClientInterface {
+    fetchTransactionDetails(txHash: string): Promise<SolanaTransaction>
+}
+
+export class SolanaTransactionClient implements SolanaTransactionClientInterface{
+
+    /**
+     * Fetches detailed transaction data from Helius API
+     * @param txHash - The transaction hash to fetch details for
+     * @returns Promise<SolanaTransaction> - The transaction details from Helius API
+     */
+    async fetchTransactionDetails(txHash: string): Promise<SolanaTransaction> {
+        const networkResponse = await fetch(`https://api.helius.xyz/v0/transactions?api-key=${appConfig.heliumAPIKey}`, {
+          method: "POST",
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            transactions: [txHash]
+          })
+        })
+        let data: SolanaTransaction[] = await networkResponse.json()
+        return data[0]
+    }
+}
+
+/**
+ * Determines if a SOL amount is considered "dust" (very small amount used in dusting attacks)
+ * @param amount - The SOL amount to check (in lamports)
+ * @returns boolean - True if the amount is considered dust, false otherwise
+ */
+function isDust(amount: number): boolean {
```

Comment:
If this function comes under the responsibility of this client then please move it as a class method. 
If it is purely a utility function then please move out it from this file.